.evergreen/scaffold_atlas.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import json
+import logging
+import os
+from pathlib import Path
+from typing import Any, Union
+
+from pymongo import MongoClient, database
+from pymongo.results import InsertManyResult
+
+logging.basicConfig()
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.DEBUG if os.environ.get("DEBUG") else logging.INFO)
+
+DATABASE_NAME = os.environ.get("DATABASE")
+CONN_STRING = os.environ.get("CONN_STRING")
+REPO_NAME = os.environ.get("REPO_NAME")
+DIR = os.environ.get("DIR")
+TARGET_DIR = os.environ.get("TARGET_DIR")
+DB_PATH = "database"
+
+
+def upload_data(client: database, filename: Path) -> None:
```

Comment:
Should this be `Database`? Or is it supposed to be a `MongoClient`?

Reply:
it's `Database`. 
The double `database` usage is confusing, so I can opt to also make it `db`.